### PR TITLE
Sync gentoo/rust with Heather/gentoo-rust.

### PR DESCRIPTION
**Pull request by omasanori**
I found that the gentoo/rust overlay is not up-to-date. Well, in fact, I've forgotten this repository. :disappointed:

@Heather @jauhien Shall we import issues into gentoo/rust and move further development to there? It may lead to confusion that we don't treat the repository in the gentoo organization as upstream, but the initial developer's.

### CODE REVIEW

**Comment by Heather** [comment]
@omasanori there should not be merge, just push

**Comment by Heather** [comment]
@omasanori about moving to this repository - I'm fully agreed, after pushing changes to this repository we should change primary address in layman.

**Comment by omasanori** [comment]
OK. Could you push them? I don't have push permission.

**Comment by Heather** [comment]
@omasanori yes but when I will be near my netbook
